### PR TITLE
add approxposterior to conda forge

### PR DESCRIPTION
**Pull request by dflemin3**

### CODE REVIEW

File: recipes/approxposterior/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pytest
+    - python
+    - pip
+    - numpy
+    - matplotlib
```
Comment (CJ-Wright) [inline recipes/approxposterior/meta.yaml]: Is this a build dep?

File: recipes/approxposterior/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  description: |
+    This package is a Python implementation of Bayesian Active Learning for Posterior Estimation by Kandasamy et al. (2015) and Adaptive Gaussian process approximation for Bayesian inference with expensive likelihood functions by Wang & Li (2017). These algorithms allows the user to compute approximate posterior probability distributions using computationally expensive forward models by training a Gaussian Process (GP) surrogate for the likelihood evaluation. The algorithms leverage the inherent uncertainty in the GP's predictions to identify high-likelihood regions in parameter space where the GP is uncertain. The algorithms then run the forward model at these points to compute their likelihood and re-trains the GP to maximize the GP's predictive ability while minimizing the number of forward model evaluations. Check out Bayesian Active Learning for Posterior Estimation by Kandasamy et al. (2015) and Adaptive Gaussian process approximation for Bayesian inference with expensive likelihood functions by Wang & Li (2017) for in-depth descriptions of the respective algorithms.
```
Comment (CJ-Wright) [inline recipes/approxposterior/meta.yaml]: Could you please break this up over multiple lines?

File: recipes/approxposterior/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  doc_url: https://dflemin3.github.io/approxposterior/
+  dev_url: https://github.com/dflemin3/approxposterior
+
+extra:
+  recipe-maintainers:
+    - dflemin3
+    - jakevdp
```
Comment (CJ-Wright) [inline recipes/approxposterior/meta.yaml]: @jakevdp are you ok with being maintainer?

Comment (jakevdp) [reply recipes/approxposterior/meta.yaml]: Yep!

File: recipes/approxposterior/meta.yaml (hunk 1 above)
Comment (dflemin3) [reply recipes/approxposterior/meta.yaml]: Good catch! We pruned the builds deps to just include python.

File: recipes/approxposterior/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```
[the remark this answers is not in the document]
Comment (jakevdp) [reply recipes/approxposterior/meta.yaml]: What does host mean in this context?

[the remark this answers is not in the document]
Comment (mariusvniekerk) [reply recipes/approxposterior/meta.yaml]: Build is basically for compilers and such. Host is typically the place for build time dependencies like python and other c libraries.

[the remark this answers is not in the document]
Comment (dflemin3) [reply recipes/approxposterior/meta.yaml]: Got it, thanks!  Changed build to host.

**Pull request merged**